tests/lib/rule-tester/rule-tester.js

```diff
@@ -2628,45 +2628,50 @@ describe("RuleTester", () => {
         });
     });
 
-    it("should allow subclasses to set the describe/it/itOnly statics and should correctly use those values", () => {
-        const assertionDescribe = assertEmitted(ruleTesterTestEmitter, "custom describe", "this-is-a-rule-name");
-        const assertionIt = assertEmitted(ruleTesterTestEmitter, "custom it", "valid(code);");
-        const assertionItOnly = assertEmitted(ruleTesterTestEmitter, "custom itOnly", "validOnly(code);");
+    describe("Subclassing", () => {
```

Comment:
While copying over I noticed this wasn't inside of a `describe` block so I added one.